lms/lmsdb/models.py

```diff
@@ -131,6 +131,17 @@ def is_viewer(self) -> bool:
         return self.name == RoleOptions.VIEWER.value or self.is_manager
 
 
+class Course(BaseModel):
+    name = CharField(unique=True)
+    date = DateTimeField()
+    due_date = DateTimeField(null=True)
+    is_finished = BooleanField(default=False)
+    close_registration_date = DateTimeField(default=datetime.now)
```

Comment:
Can you please add autogenerated field of `invite_code`?

I think about sending a registration link to users - directly into a specific course :)